backend/fms_core/template_importer/row_handlers/sample_selection_qpcr/sample.py

```diff
@@ -12,7 +12,7 @@ class SampleSelectionQPCRRowHandler(GenericRowHandler):
     def __init__(self):
         super().__init__()
 
-    def process_row_inner(self, sample, sample_information, process_measurement, process_measurement_properties):
+    def process_row_inner(self, sample, process_measurement, process_measurement_properties):
```

Comment:
Not sure this is the right way to do this every process_row_inner has a sample_information. We expect it to have one so if we call the function and pass parameter by position there is a high risk of error. Just pass an empty object if you want. but leave the method signature the same.